docs/about/themes-translation.md

```diff
@@ -0,0 +1,147 @@
+# MkDocs themes translation
+
+How to add a locale translation to mkdocs built-in themes.
```

Comment:
At first I thought it was strange that this file was under `about`, but I suppose this is more like a contributing guide (specific to translation) than it is a user guide. So, we can leave it here.

However, it might make sense to have the Contributing Guide link back to this. Something along the lines of: To contribute translations see...